### ISSUE TITLE
Trigger from icon using classes instead of IDs

### ISSUE THREAD

**Issue by moshner** [mydea/PikadayResponsive#12]
I'm trying to get Pikaday to trigger from the icon that I have absolutely positioned inside the input field. To compound this situation I'm also trying to write it to always trigger from classes instead of ids. We frequently have several dates that need to be picked on a page and I want to keep the javascript to a minimum. 

How would I trigger the pikaday action from the icon as well as the box?

Take a look: http://codepen.io/moshner/pen/rLQXjY

**Comment by mydea** [comment]
This is a bit tricky, because native date pickers can't really be opened programmatically.
However, I just released v0.7.0, which will now automatically add IDs to your generated inputs (either the pikaday input or the native date field), in the format `originalId-input`. You can then trigger the date picker from anywhere on your page with a simple label. See https://mydea.github.io/PikadayResponsive/ at the bottom for an example.

So basically:

```html
<input id='my-date-field'>

<!-- Something else -->\

<label for='my-date-field-input'>Open Date Picker!</label>
```

```js
pikadayResponse(document.getElementById('my-date-field'));
```

I tried it in Chrome & Android, and it works there. If you run into any problems, please feel free to reopen this issue!
